### PR TITLE
Create a new activity and home screen for a pet owner

### PR DESCRIPTION
**Pull request by yairMar1**
Veterinarian Registration Activity

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -10,4 +10,46 @@
     <string name="invalid_password">Password must be >5 characters</string>
     <string name="login_failed">"Login failed"</string>
     <string name="title_activity_pet_sign_up">pet_signUp</string>
+    <!-- Strings used for fragments for navigation -->
```
Comment (meshs1) [inline app/src/main/res/values/strings.xml:13]: These strings can also be deleted

**Review by meshs1 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/myvet/myvet/SecondPage.kt
[the remark this answers is not in the document]
Comment (yairMar1) [reply app/src/main/java/com/myvet/myvet/SecondPage.kt:1]: You are right.
Initially this page was designed as a user homepage but as we went along, we separated it into two pages. One is a homepage for veterinarians and the other is for pet owners.

File: app/src/main/java/com/myvet/myvet/SignUp.kt [hunk 2]
```diff
@@ -36,6 +36,13 @@ class SignUp : AppCompatActivity() {
             startActivity(intent)
             finish()
         }
+
+        Vet_Btn.setOnClickListener {
+            Log.i("Test pet", "The user will go to pet sign up page")
```
[the remark this answers is not in the document]
Comment (yairMar1) [reply app/src/main/java/com/myvet/myvet/SignUp.kt:41]: I updated the printing of log - for veterinarian registration.
I also changed the names of the button variables to JAVA convention

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (yairMar1) [reply app/src/main/res/values/strings.xml:13]: I deleted these strings.

File: app/src/main/res/navigation/nav_graph.xml
[the remark this answers is not in the document]
Comment (yairMar1) [reply app/src/main/res/navigation/nav_graph.xml:1]: I deleted this file.

**Review by meshs1 — APPROVED** [review]

**Pull request merged**